protocols/convert_IDAT_To_GTC_Format.sh

```diff
@@ -0,0 +1,51 @@
+#MOLGENIS walltime=05:59:00 mem=8gb ppn=1
+
+#string iaapVersion
+#string bpmFile
+#string egtFile
+#string egt
+#string SentrixBarcode_A
+#string convertDir
+#string IDATFilesPath
+#string logsDir
+#string Project
+#string runID
+#string resultDir
+#string GTCFilesPath
+
+module load "${iaapVersion}"
+module list
+
+mkdir -p "${convertDir}/"
+mkdir -p "${convertDir}/${SentrixBarcode_A}"
+
+##Command to convert IDAT files to GTC files
+${EBROOTIAAP}/iaap-cli/iaap-cli gencall "${bpmFile}" "${egtFile}" "${convertDir}/${SentrixBarcode_A}" -f "${IDATFilesPath}/${SentrixBarcode_A}" -g
+
+## md5sum the output
+
+for gtcfile in $(ls "${convertDir}/${SentrixBarcode_A}/"*.gtc)
+do
+	md5sum "${gtcfile}" > "${gtcfile}".md5
+done
+
+#Move GTC files and md5's to rawdata/array/GTC/ folder
+mkdir -p "${resultDir}"
```

Comment:
ik zou hier ook ${SentrixBarcode_A} aan toevoegen zodat je in de move stap hierna dit kunt doen:
mv "${convertDir}/${SentrixBarcode_A}/"* "${GTCFilesPath}/${SentrixBarcode_A}/"
in het geval van 2x runnen gaat anders je code stuk omdat ie dit zegt: "directory not empty"